rdkit/Chem/UnitTestPeriodicTable.py

```diff
@@ -0,0 +1,29 @@
+# $Id$
+#
+#  Copyright (C) 2003-2006 Rational Discovery LLC
```

Comment:
Would be useful to have a file with the default blurb to be used in these files.